dev.opam

```diff
@@ -37,12 +37,15 @@ depends: [
   "caqti-driver-postgresql"
   "caqti-lwt"
   "caqti-async"
+  "caqti-eio"
+  "eio_main"
   "ppx_rapper"
 
   # Below is for testing only
   "base"
   "ppx_assert"
   "ppx_inline_test"
+  "utop"
 
   # Below is for documentation purposes
   "mdx"
```

Comment:
If this was the OPAM file of a real release, the test dependencies would have `{with-test}` attributes and the `utop` would be omitted.  Should we aim for convenience or a more authentic opam file?

Reply:
Thanks for the suggestion.

I will create a PR to improve this file + automate setting up the opam switch+running the tests so that we know everything works for specific dependency versions.